tasks/hough-transform/src/main.rs

```diff
@@ -118,10 +125,33 @@ fn hough(image: &ImageGray8, out_width: usize, out_height: usize) -> ImageGray8
     accum
 }
 
+fn show_image(image: &ImageGray8, title: &str) {
+    let mut window = Window::new(title, image.width, image.height, WindowOptions::default())
+        .unwrap_or_else(|e| {
+            panic!("{}", e);
+        });
+
+    // Limit to max ~60 fps update rate
+    window.limit_update_rate(Some(std::time::Duration::from_micros(16600)));
+
+    // load image
+    let u32_buffer: Vec<u32> = image
+        .data
+        .iter()
+        // .chunks(3)
+        .map(|v| (u32::from(*v) << 16) | (u32::from(*v) << 8) | u32::from(*v) as u32)
+        .collect();
+
+    while window.is_open() && !window.is_key_down(Key::Escape) {
+        window
+            .update_with_buffer(&u32_buffer, image.width as usize, image.height as usize)
+            .unwrap();
```

Comment:
propagate these unwraps.

Reply:
After we remove the windowing with minifb, then it's not needed :)